modules/codegen/src/main/scala/com/twilio/guardrail/swagger/Escape.scala

```diff
@@ -0,0 +1,82 @@
+package com.twilio.guardrail.swagger
+
+import scala.meta.{ Init, Term, Tree, Type }
+import scala.util.matching.Regex
+
+object Escape {
```

Comment:
All the manual tree escaping was removed, this logic is all handled by scalameta properly now.